config/samples/kafka-frameworkversion.yaml

```diff
@@ -242,23 +242,23 @@ spec:
       apiVersion: apps/v1
       kind: StatefulSet
       metadata:
-        name: {{FRAMEWORK_NAME}}
-        namespace: {{NAMESPACE}}
+        name: {{ .FrameworkName }}
```

Comment:
These non-parameter varaibles should be layed out in documentation somewhere

Reply:
This is part of KEP-9's Execution State section. I'll open a separate PR updating KEP-9 once the dust settles on that and we get your PR + new install method all in.

Reply:
(Then, we also need to write actual docs based on that section)